homework-solutions/js/hw-4/task-3.js

```diff
@@ -6,3 +6,26 @@
   преобразовываясь в number
 
 */
+
+let minAge, maxAge, age;
+minAge = 18;
+maxAge = 60;
+age = '19';
+
+if (typeof age === 'string') {
+  age = parseInt(age);
+}
+
+if (typeof age !== 'number') {
+  console.log('Incorrect data type');
+} else {
+  if (age < minAge) {
```

Comment:
В данном случае `parseInt` не совсем подходит, он вьітащит число, но изначальное значение может не подойти, например, `34wd`

Вьіполняем проверку, если не подходит одно действие, если подходит - приводим к числу и проходим по ветвлениям
Неплохо узнать разницу между `isNaN` и `Number.isNaN` 

```suggestion
if (isNaN(age)) {
  console.log('Incorrect data type');
} else {
  age = +age; // Number(age)
  if (age < minAge) {
```